Shipr-Kubernetes/shipr-inventory-deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: shipr-inventory
+  labels:
+    app: shipr-inventory
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: shipr-inventory
+  template:
+    metadata:
+      labels:
+        app: shipr-inventory
+    spec:
+      containers:
+      - name: shipr-inventory
+        image: osamaazm/test-repo:shipr-inventory
+        ports:
+        - containerPort: 8000
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: 8000
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          failureThreshold: 3
+        livenessProbe:
+          httpGet:
+            path: /health
+            port: 8000
+          initialDelaySeconds: 10
+          periodSeconds: 15
+          failureThreshold: 3
```

Comment:
I don't think the application offers a /health endpoint, how did you come to this conclusion? Plus did you test this ?

Reply:
Changed

Reply:
I wanted to know your thought process so that we may fix any misunderstanding which you may have. Resolving this for now.